Source/ActionController.swift

```diff
@@ -179,13 +179,29 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
             section.actions.append(action)
         }
     }
-
     @discardableResult
     open func addSection(_ section: Section<ActionDataType, SectionHeaderDataType>) -> Section<ActionDataType, SectionHeaderDataType> {
         _sections.append(section)
         return section
     }
     
+    public typealias ClosureAction = ((Action<ActionDataType>) -> Bool)
+    
+    open func removeAction(closure: ClosureAction) {
+        if let section = _sections.last {
+            guard let index = section.actions.index(where: { (item) -> Bool in
+                closure(item)
+            }) else { return }
+            section.actions.remove(at: index)
+        }
+    }
+    
+    open func removeAction(_ actionPosition: Int) {
+        if let section = _sections.last {
```

Comment:
I totally forgot about sections, you can have many sections in one action controller, so you have to indicate the section and the row for the action that you want to remove. So maybe would be better to receive an IndexPath here instead of only an index.